Starter notebook for mission, not complete yet

ohinson01 opened the pull request.

Shaheerairaj reviewed (changes requested):
Hi Olivia, could you include the checkpoints file in your gitignore folder and send another PR without the checkpoint please?

ohinson01 commented:
HI Shaheer. I went ahead and deleted the checkpoints file. Hopefully the next time I perform a PR, it will ignore it. I added it to my gitignore file, so hope it works next time.
